types/trouter/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for trouter 2.0
+// Project: https://github.com/lukeed/trouter
+// Definitions by: Markus Lanz <https://github.com/stahlstift>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+declare namespace Trouter {
```

Comment:
Any reason we need to use a namespace here?

Reply:
No - I thought it's "best practice". I will remove it.

Reply:
It's only really needed when you have to use declaration merging eg. the module you're trying to define is using `module.exports = ...`